src/elements/content-sidebar/SidebarNav.js

```diff
@@ -94,6 +113,112 @@ const SidebarNav = ({
             focusPrompt();
         }
     };
+    const { icon: CustomTabIcon, index: customTabIndex = 0, title: customTabTitle, navButtonProps } = customTab || {};
+    const customTabId = customTab && typeof customTab.id === 'string' ? customTab.id.trim() : '';
+    const hasCustomTab = !!customTabId;
+    const hasBoxAICustomTab = customTabId === 'boxai';
```

Comment:
I think you can remove the logic and let the consumer handle these things. for example, `hasBoxAICustomTab` isn't needed here because you can control this using in EUA by just adjusting the value of `useFeatureConfig('boxai.sidebar')`. that way this component doesn't need to know about specific use cases and business logic